### PR TITLE
Feat/report input

### PR DESCRIPTION
**Pull request by dreymoreau**
- added styling to the report page
- created form
- fixed styling of blog posts on main

### CODE REVIEW

File: src/styles/report.css [hunk 1]
```diff
@@ -30,3 +34,37 @@ h1 {
...
+	margin-top: 20px;
+}
+
+button[type="submit"]{
+	background: #ED342F;
+}
+
+button[type="submit"]:hover{
+	background: #A8569A;
+}
```
Comment (madcampos) [inline src/styles/report.css:70]: Let's namespace the selectors to `#report-form` (just a suggestion for the name), that way the styles don't "leak" everywhere else.

Also, moved the `input` and `textarea` styles to the namespace using the [`:is` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) to make life easier.

```suggestion
#report-form {
  border-radius: 5px;
  border: 5px solid #ED342F;
  padding: 20px;
  width: 75%;
  margin: 0 auto;
}

#report-form :is(input, textarea) {
  width: 100%;
  padding: 12px;
  border: 1px solid #cccccc;
  border-radius: 4px;
  resize: vertical;
}

#report-form label {
  padding: 12px 12px 12px 0;
  display: inline-block;
}

#report-form input {
	margin-block-start: 20px;
}

#report-form button[type="submit"]{
	background: #ED342F;
}

#report-form button[type="submit"]:hover{
	background: #A8569A;
}
```

File: src/layouts/BlogPost.astro [hunk 2]
```diff
@@ -7,6 +7,7 @@ import PostHeader from '../components/PostHeader/index.astro';
 import SkipToContent from '../components/SkipToContent/index.astro';
 import TagList from '../components/TagList/index.astro';
 import BaseLayout from './Base/index.astro';
+import './styles.css';
```
[the remark this answers is not in the document]
Comment (dreymoreau) [reply src/layouts/BlogPost.astro:10]: ou yes!!!!! this is definitely a good idea!!

File: src/components/PostCard/index.astro [hunk 3]
```diff
@@ -21,7 +23,7 @@ const formatter = new Intl.DateTimeFormat('en-US', { dateStyle: 'long', timeStyl
 ---
 <article
 	class:list={[
-		'post-card bg-gradient-to-t rounded-3xl p-4 hover:from-yellow to-transparent grid gap-2 shadow-md',
+		'container post-card bg-gradient-to-t rounded-3xl p-4 hover:from-yellow to-transparent grid gap-2 shadow-md',
```
[the remark this answers is not in the document]
Comment (dreymoreau) [reply src/components/PostCard/index.astro:26]: omg yes totally wow

File: src/components/PageHeader/styles.css [hunk 4]
```diff
@@ -1,3 +1,7 @@
+.center-h1 {
+	text-align: center;
+}
+
 #page-header {
 	width: auto;
 	margin: 1.25rem auto;
```
[the remark this answers is not in the document]
Comment (dreymoreau) [reply src/components/PageHeader/styles.css:7]: keeps code dry!! :) awesome

File: src/components/PostCard/styles.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+	margin-top: 20px;
+}
+
+button[type="submit"]{
+	background: #ED342F;
+}
+
+button[type="submit"]:hover{
+	background: #A8569A;
+}
```
[the remark this answers is not in the document]
Comment (dreymoreau) [reply src/components/PostCard/styles.css:32]: OU!! yes. you are correct! good catch

File: src/layouts/BlogPost.astro (hunk 2 above)
[the remark this answers is not in the document]
Comment (dreymoreau) [reply src/layouts/BlogPost.astro:10]: added to our ongoing big push issue :) so we can reference tmo during our chat :))

File: src/styles/report.css [hunk 6]
```diff
@@ -1,4 +1,8 @@
 
+* {
+  box-sizing: border-box;
+}
+
```
[the remark this answers is not in the document]
Comment (dreymoreau) [reply src/styles/report.css:5]: oh! yes yes :) its in base.css!! will remove this

File: src/styles/report.css (hunk 1 above)
Comment (dreymoreau) [reply src/styles/report.css:70]: this is so awesome!!! definitely is way more specific! thank you!!

**Review by madcampos — APPROVED** [review]

**Pull request merged**